configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
ok. Definitely let us try the short term solution first. Let us know when this is ready for a temporary merge: @xylar, maybe we could start merging this to master_v02 and test it in the PreAndPostProcessing framework? what do you think?